docker/actinia-core-dev/Dockerfile

```diff
@@ -9,8 +9,8 @@ ENV FLASK_ENV=development
 ENV FLASK_DEBUG=1
 ENV DEFAULT_CONFIG_PATH=/etc/default/actinia
 
-RUN pip3 uninstall actinia-core -y
-RUN pip3 uninstall actinia_core -y
+RUN pip3 uninstall actinia-core -y \
+    RUN pip3 uninstall actinia_core -y
```

Comment:
Why is it like this and not like in the test-Dockerfile like ` &&  \`?

Reply:
good catch, updated it.